relayer/src/chain/cosmos/query/status.rs

```diff
@@ -0,0 +1,36 @@
+use ibc::core::ics24_host::identifier::ChainId;
+use ibc::Height;
+use tendermint_rpc::{Client, HttpClient, Url};
+
+use crate::chain::ChainStatus;
+use crate::error::Error;
+
+pub async fn query_status(
+    chain_id: &ChainId,
+    rpc_client: &HttpClient,
+    rpc_address: &Url,
+) -> Result<ChainStatus, Error> {
```

Comment:
This is refactored from:

https://github.com/informalsystems/ibc-rs/blob/3e34b4b916eb91f680412e5fdd3ea045a99526d2/relayer/src/chain/cosmos.rs#L907-L916

https://github.com/informalsystems/ibc-rs/blob/3e34b4b916eb91f680412e5fdd3ea045a99526d2/relayer/src/transfer.rs#L98-L111